nuxt.config.ts

```diff
@@ -115,6 +114,7 @@ export default defineNuxtConfig({
       translateApi: '',
       // Use the instance where Elk has its Mastodon account as the default
       defaultServer: 'm.webtoo.ls',
+      singleInstance: false,
```

Comment:
shouldn't use `process.env.NUXT_PUBLIC_SINGLE_INSTANCE` here?

Reply:
Yes, Nuxt will automatically override it.